include/nanoeigenpy/decompositions/ldlt.hpp

```diff
@@ -93,9 +93,8 @@ void exposeLDLTSolver(nb::module_ m, const char *name) {
 
           .def(
               "compute",
-              [](Solver &c, VectorType const &matrix) -> Solver & {
-                return c.compute(matrix);
-              },
+              [](Solver &c, Eigen::EigenBase<MatrixType> const &matrix)
```

Comment:
This will not work as there is no automatic downcasting from `Eigen::EigenBase<T>` to `T` implemented in nanobind.